numpy/lib/tests/test_format.py

```diff
@@ -934,6 +936,26 @@ def test_unicode_field_names(tmpdir):
         with assert_warns(UserWarning):
             format.write_array(f, arr, version=None)
 
+def test_header_growth_axis():
+    import io
```

Comment:
There's already a `from io import BytesIO` import at the top level. No need for a local import.

Reply:
Done